src/main/markdown/community.md

```diff
@@ -34,7 +34,7 @@ Before submitting your post, please try the following:
 
 ## Other communities
 
-There is a [GWT community over at Google+](https://plus.google.com/u/0/communities/116543000751323604177). There's also the IRC channel #gwt on the Freenode network.
+There is a [GWT Gitter Community](https://gitter.im/gwtproject/gwt). There's also the IRC channel #gwt on the Freenode network.
```

Comment:
The irc channel is actually `##gwt` - this is a `#gwt`, but it is private.

Reply:
Ok, this is a separate issue right? If IRC channel ist still active we should change this to public channel 👍

Reply:
> The irc channel is actually `##gwt` - this is a `#gwt`, but it is private.

Fixed with https://github.com/gwtproject/gwt-site/pull/304